### ISSUE TITLE
Could you explain to me how to install and configure it correctly in fedora 31?

### ISSUE THREAD

**Issue by xzj8b3** [rapid7/metasploit-framework#12371]
Could you explain to me how to install and configure it correctly in fedora 31, is there a repository and an updated guide for such a system?

**Comment by strasharo** [comment]
Install rvm and run it through it:
https://rvm.io/rvm/install

**Comment by exploide** [comment]
Since Fedora 31 is not released yet, I'm using Fedora 30 instead, but that should make no big difference.

I recommend using rapid7's RPM repository to install the (nightly) version(s) of metasploit-framework. You can find the explanation in the wiki: https://github.com/rapid7/metasploit-framework/wiki/Nightly-Installers

There, you find the install instructions telling you to download and execute that script. But you can also look into the script and perform the necessary steps manually: https://raw.githubusercontent.com/rapid7/metasploit-omnibus/master/config/templates/metasploit-framework-wrappers/msfupdate.erb

Basically it add the RPM repository and then installs metasploit-framework.

**Comment by xzj8b3** [comment]
> Installa rvm ed eseguilo:
> https://rvm.io/rvm/install

At least tell me how to install it correctly

**Comment by xzj8b3** [comment]
> Installa rvm ed eseguilo:
> https://rvm.io/rvm/install

Ultima verifica della scadenza dei metadati: 0:07:23 fa il sab 28 set 2019, 17:37:07.
Nessuna corrispondenza per l'argomento: msfinstall

**Comment by xzj8b3** [comment]
> Installa rvm ed eseguilo:
> https://rvm.io/rvm/install

Explain to me how to install it correctly ..Metasploit installed, but missing Ruby and PostgreSQL 

I never installed it on a fedora.

**Comment by jmartin-r7** [comment]
This question is answered simplest above by @exploide.  

The only expansion to offer is to consider instructions here for a [development environment](https://github.com/rapid7/metasploit-framework/wiki/Setting-Up-a-Metasploit-Development-Environment)

While the instructions are for a Debian based OS they will be very similar for an RPM based install. Since you are referring to not having `Ruby and PostgreSQL` this may be the route you need.

**Comment by xzj8b3** [comment]
> This question is answered simplest above by @exploide.
> 
> The only expansion to offer is to consider instructions here for a [development environment](https://github.com/rapid7/metasploit-framework/wiki/Setting-Up-a-Metasploit-Development-Environment)
> 
> While the instructions are for a Debian based OS they will be very similar for an RPM based install. Since you are referring to not having `Ruby and PostgreSQL` this may be the route you need.

I will wait for Fedora 31 to be at least in final release
